.github/workflows/build-test-e2e.yml

```diff
@@ -81,8 +82,8 @@ jobs:
         run: colima stop
   merge-reports:
     # Merge reports after playwright-tests, even if some shards have failed
-    # But skip this job if the previous job was cancelled
-    if: ${{ !cancelled() }}
+    # But skip this job if the previous job was cancelled or skipped
+    if: ${{ !cancelled() && needs.playwright-tests.result != 'skipped' }}
```

Comment:
"skipped" !== "cancelled" -- skip the job if the workflow was "skipped", as in this [case](https://github.com/Sage-Bionetworks/SynapseWebClient/actions/runs/6912914333).